request_handlers.cc

```diff
@@ -0,0 +1,37 @@
+#include "request_handlers.h"
+
+#include <string>
+#include <iostream>
+
+using namespace lightning_server;
+
+const int MAX_REQ_SIZE = 8192;
```

Comment:
Addressed as part of https://github.com/UCLA-CS130/Mr.-Robot-et-al./pull/15/commits/83b15ef868d5dc7f6c0b4da4df3f155f05d6af12